scanpullrequest/scanpullrequest.go

```diff
@@ -290,10 +309,10 @@ func checkoutToCommitAtTempWorkingDir(scanDetails *utils.ScanDetails, commitHash
 	return gitManager.CheckoutToHash(commitHash, wd)
 }
 
-func getAllIssues(cmdResults *results.SecurityCommandResults, allowedLicenses []string, hasViolationContext bool) (*utils.IssuesCollection, error) {
+func getAllIssues(cmdResults *results.SecurityCommandResults, allowedLicenses []string, includeVulnerabilities, hasViolationContext bool) (*issues.ScansIssuesCollection, error) {
 	log.Info("Frogbot is configured to show all vulnerabilities")
 	simpleJsonResults, err := conversion.NewCommandResultsConvertor(conversion.ResultConvertParams{
-		IncludeVulnerabilities: true,
+		IncludeVulnerabilities: includeVulnerabilities,
```

Comment:
General question: why is this up to a parameter if we want to get all issues?
do you refer to a case where we want all found violations? I dont understand this change - this is not a new field, so why we didnt use it so far?